src/HostAgnostic.props

```diff
@@ -17,6 +17,8 @@
     <!-- Roslyn -->
     <PackageReference Include="Microsoft.Net.RoslynDiagnostics" />
     <PackageReference Include="Microsoft.VisualStudio.LanguageServices" />
+    <PackageReference Include="Microsoft.CodeAnalysis" />
```

Comment:
Really - we didn't use to have a reference to all of Roslyn - why do we need it now?  For `ProjectId`?

Reply:
LanguageSevices changed its dependencies; before we had access to Microsoft.CodeAnalysis.CSharp/Microsoft.CodeAnalysis.VisualBasic (parsers) and the associated language-specific workspace APIs (rename) through it - now we have to explicitly state it.